xl_auth/commands.py

```diff
@@ -608,3 +609,11 @@ def _get_manually_deleted_permissions():
                   % (permission.user.email, permission.collection.code, wipe_permissions))
             if wipe_permissions:
                 permission.delete()
+
+
+@click.command()
+def prod_run():
+    from sh import Command
+    flask_cmd = Command('flask')
+    click.echo(flask_cmd('db', 'upgrade').stderr)
+    execlp('gunicorn', 'gunicorn', '-c', 'gunicorn_conf.py', 'autoapp:app')
```

Comment:
I haven't had time to try it yet, but I hate the config file... One unfortunate extra layer of complexity, methinks. Supporting environment variables, command-line options AND config file options. Could we please remove it and pass number of workers using Gunicorn's builtin `WEB_CONCURRENCY=N` envvar? Or, if undesirable for some reason, adding it as a Click option instead?

Reply:
I actually kinda like the config file, since it is clearly separated from everything else. That is, gunicorn isn't really a part of the application and having a dedicated config file for it is (to me, at least) saying that "This is the only place where we put gunicorn configurations".

Let's discuss it today.